cves/kernel/CVE-2021-29650.yml

```diff
@@ -358,8 +411,13 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    The primary issue was that a data race in the netfilter subsystem could lead
+    to a null pointer dereference during replacement of a table. 
+    Only users with the CAP_NET_ADMIN user role in any namespace could use
```

Comment:
I think you should add this part to the description of the vulnerability. It is unclear that only certain users could cause a denial of service attack.

Reply:
I appreciate the suggestion.